### DISCUSSION TITLE
Form override not useful

### DISCUSSION THREAD

**Discussion by ischaojie** [aminalaee/sqladmin#227] [Q&A]
I try to use `RadioField` instead `SelectField` for enum column, But using `form_overrides` does not work.

```
status = Column(Enum(Status))
```

```
class UserAdmin(ModelAdmin, model=User):
    column_list = [User.id, User.name, User.age]
    form_overrides = dict(status=RadioField)
```

Result:
<img width="1586" alt="image" src="https://user-images.githubusercontent.com/12986082/176987727-68ac4310-d8cb-41d0-a036-e58f48d7039c.png">

The reason is `form_overrides` just return the basic RadioField, with no choices.

So there is a better way to realize it?

**Comment by aminalaee** [comment]
I think you also need to pass the `form_args` related fields:

```python
class UserAdmin(ModelAdmin, model=User):
    form_overrides = dict(status=RadioField)
    form_args = dict(status=dict(choices=[True, False]))
```

But the UI is not working correctly, feel free to create a PR for it.
